CHANGELOG.md

```
@@ -8,6 +8,33 @@
+- [Add CodeQL for vulnerability scan](https://github.com/espressif/vscode-esp-idf-extension/pull/1912)
+- [Remove Project Configuration Editor in favor of CMakePresets](https://github.com/espressif/vscode-esp-idf-extension/pull/1842)
+
+## Bug Fixes
+
+- [Pick adf_examples if present else fallback to examples](https://github.com/espressif/vscode-esp-idf-extension/pull/1848)
+- [Fix preserve user custom idf eimExecutableArgs](https://github.com/espressif/vscode-esp-idf-extension/pull/1844) Thanks @jiesou !
+- [Fix avoid waiting terminal focus over Remote SSH](https://github.com/espressif/vscode-esp-idf-extension/pull/1831)
+- [Use BASH as default terminal for activation script](https://github.com/espressif/vscode-esp-idf-extension/pull/1853)
+- [Bump CI ESP-IDF version to 6.0.1](https://github.com/espressif/vscode-esp-idf-extension/pull/1878)
+- [Fix telemetry session scope deduplication](https://github.com/espressif/vscode-esp-idf-extension/pull/1886)
+- [Use python3 binary name consistently ](https://github.com/espressif/vscode-esp-idf-extension/pull/1840)
```

markdownlint-cli2 (0.23.2)

[warning] Spaces inside link text

(MD039, no-space-in-links)